docs/units/logarithmic_units.rst

```diff
@@ -137,6 +137,35 @@ appropriate equivalency::
     >>> V.to(u.ABmag, u.spectral_density(5500.*u.AA))  # doctest: +FLOAT_CMP
     <Magnitude [16.99023831, 21.10978201, 22.50053827] mag(AB)>
 
+This is particularly useful for converting magnitude into flux density.
+``V`` is currently in ST magnitudes, which is based on flux densities per
+unit wavelength::
+
+    >>> flam = V.to(u.erg/u.s/u.cm**2/u.AA)
+    >>> flam  # doctest: +FLOAT_CMP
+    <Quantity [5.75439937e-16, 1.29473986e-17, 3.59649961e-18] erg / (Angstrom cm2 s)>
+
+To convert ``V`` to flux density per unit frequency, we again need the 
+appropriate equivalency::
+
+    >>> lam = 5500 * u.AA
+    >>> fnu = V.to(u.erg/u.s/u.cm**2/u.Hz, u.spectral_density(lam))
+    >>> fnu  # doctest: +FLOAT_CMP
+    <Quantity [5.80636959e-27, 1.30643316e-28, 3.62898099e-29] erg / (cm2 Hz s)>
+
+We could have also used frequency for the equivalency::
+
+    >>> nu = 5.45077196e+14 * u.Hz
+    >>> fnu = V.to(u.erg/u.s/u.cm**2/u.Hz, u.spectral_density(nu))
+    >>> fnu  # doctest: +FLOAT_CMP
+    <Quantity [5.80636959e-27, 1.30643316e-28, 3.62898099e-29] erg / (cm2 Hz s)>
+
+.. Note::
+
+    When converting magnitudes to flux densities, order of operations matters.
+    For example, ``21 * u.ABmag.to(u.erg/u.s/u.cm**2/u.Hz)`` is not the same as 
```

Comment:
In order to convert an AB mag of 21 to <something else>, the value and the unit need to be established before the conversion. For example, 21 * blah blah will give you 21 times an AB mag of 1 while (21*u.ABmag) in <blah units> will give you ABmag of 21 in <blah units>.